### PR TITLE
[49863] Update of texts

### PR DESCRIPTION
**Pull request by Frederik-S-N-dk**
Updated texts in the app

### CODE REVIEW

File: NDB.Covid19/NDB.Covid19/Locales/en.json [hunk 1]
```diff
@@ -73,29 +72,27 @@
...
...st positive?’. Other app users cannot see who you are. You must log in via the ID-porten.<br><br>After you have shared your status, other users will receive a message saying that they have been nearby a person infected with COVID-19.<br><br>You can see messages in the app on the home screen. Here you can also see if you have received a new exposure message.<br><br>If Smittestopp is inactive, you cannot receive messages. You can log in and share your status, but as there is no exchange of IDs when the app is inactive, there will be nobody to share your status with. <br><br><b>The technology</b> <br>The app uses Bluetooth and is based on built-in technology developed by Apple and Google. It comprises a system for notification of exposures. You can turn this on or off in your phone’s settings. In your phone’s settings, search for ‘covid’ or ‘smittestopp’ to turn the logging on or off. <br><br>Your phone generates a random Bluetooth ID every 15 minutes. When your phone is close to another app user’s phone, the two phones will exchange IDs.<br><br> Read more about the Smittestopp app at <a href=\\\\\"www.helsenorge.no/en/smittestopp\\\\\">www.helsenorge.no/en/smittestopp</a><br><br>\"",
```
Comment (nc-kano) [inline NDB.Covid19/NDB.Covid19/Locales/en.json:75]: why so many backslashes in the link?

File: NDB.Covid19/NDB.Covid19/Locales/en.json [hunk 2]
```diff
@@ -73,29 +72,27 @@
...
   "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_3_TITLE": "The technology<br>",
-  "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_3_CONTENT": "The app uses Bluetooth and is based on built-in technology developed by Apple and Google. It comprises a system for logging exposure. You can turn this on or off in your phone’s settings.<br><br>Your phone generates a random Bluetooth ID every 15 minutes. When two phones are close to each other, they exchange IDs.<br><br> Read more about the Smittestopp app at <a href=\"https://www.helsenorge.no/en/smittestopp\">www.helsenorge.no/en/smittestopp</a><br><br>",
+  "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_3_CONTENT": "\"The app uses Bluetooth and is based on built-in technology developed by Apple and Google. It comprises a system for registering exposure. You can turn this on or off in your phone’s settings.<br><br>Your phone generates a random Bluetooth ID every 15 minutes. When two phones are nearby each other, they exchange IDs.<br><br> Read more about the Smittestopp app at <a href=\\\\\"www.helsenorge.no/en/smittestopp\\\\\">www.helsenorge.no/en/smittestopp</a><br><br>\"",
```
Comment (nc-kano) [inline NDB.Covid19/NDB.Covid19/Locales/en.json:82]: ditto

File: NDB.Covid19/NDB.Covid19/Locales/nb.json [hunk 3]
```diff
@@ -27,24 +27,24 @@
...
-  "CONSENT_THREE_PARAGRAPH_SECTION_ONE": "For å kunne ta i bruk Smittestopp må du samtykke til at Smittestopp sender og lagrer data som beskrevet ovenfor.",
-  "CONSENT_THREE_PARAGRAPH_SECTION_TWO": "Når du samtykker til å ta i bruk Smittestopp blir du i stand til å motta varsel om smittede nærkontakter. Dersom du selv skulle bli smittet velger du selv om du ønsker å varsle gjennom Smittestopp. Det er ingen som får varsel hvis du blir smittet uten at du først har godkjent dette. Du vil bli bedt om å avgi et eget samtykke for dette formålet.",
+  "WELCOME_PAGE_FIVE_TITLE": "Samtykke til aktivering av appen",
+  "CONSENT_ONE_TITLE": "Om Smittestopp ",
+  "CONSENT_ONE_PARAGRAPH_SECTION_ONE": "Det er frivillig å bruke Smittestopp. Du kan når som helst slutte å bruke Smittestopp. Du kan også når som helst slette dataene som Smittestopp har lagret på telefonen din. Smittestopp har 16 års aldersgrense. ",
+  "CONSENT_ONE_PARAGRAPH_SECTION_TWO": "Når du bruker Smittestopp, lagrer telefonen din data om andre Smittestopp-brukere du er i kontakt med. Informasjonen lagres på telefonen din i maksimalt 14 dager.  ",
```
Comment (nc-kano) [inline NDB.Covid19/NDB.Covid19/Locales/nb.json:38]: a lot of trailing white spaces in the translation.

File: NDB.Covid19/NDB.Covid19/Locales/en.json (hunk 1 above)
Comment (Frederik-S-N-dk) [reply NDB.Covid19/NDB.Covid19/Locales/en.json:75]: Hmm... don't know. It comes straight from the excel sheet @nc-kano

File: NDB.Covid19/NDB.Covid19/Locales/nb.json [hunk 4]
```diff
@@ -73,17 +72,15 @@
...
   "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_3_TITLE": "Teknologien bak<br>",
-  "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_3_CONTENT": "Appen bruker Bluetooth og er basert på en innebygd teknologi som er utviklet av Apple og Google. Teknologien er et system for varsling av smittekontakt. Du kan slå av og på denne varslingen under innstillingene på mobilen din. <br><br>Mobilen din lager en tilfeldig Bluetooth-ID hvert 15. minutt. Når to mobiler er i nærheten av hverandre, utveksler de ID-er.<br><br> Les mer om Smittestopp-appen på <a href=\"https://www.helsenorge.no/smittestopp\">www.helsenorge.no/smittestopp</a><br><br>",
+  "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_3_CONTENT": "\"Appen bruker Bluetooth og er basert på en innebygd teknologi som er utviklet av Apple og Google. Teknologien er et system for varsling av smittekontakt. Du kan slå av og på denne varslingen under innstillingene på mobilen din. <br><br>Mobilen din lager en tilfeldig Bluetooth-ID hvert 15. minutt. Når to mobiler er i nærheten av hverandre, utveksler de ID-er.<br><br> Les mer om Smittestopp-appen på <a href=\\\\\"www.helsenorge.no/smittestopp\\\\\">www.helsenorge.no/smittestopp</a><br><br>\"",
```
Comment (nc-kano) [inline NDB.Covid19/NDB.Covid19/Locales/nb.json:82]: should be https in the link

File: NDB.Covid19/NDB.Covid19/Locales/en.json (hunk 1 above)
Comment (nc-kano) [reply NDB.Covid19/NDB.Covid19/Locales/en.json:75]: so it should be corrected there. it looks like copied text from somewhere and some tool has added additional symbols. also "\ at the beginning of this field seems to be not needed

File: NDB.Covid19/NDB.Covid19/Locales/en.json [hunk 5]
```diff
@@ -73,29 +72,27 @@
...
...st positive?’. Other app users cannot see who you are. You must log in via the ID-porten.<br><br>After you have shared your status, other users will receive a message saying that they have been nearby a person infected with COVID-19.<br><br>You can see messages in the app on the home screen. Here you can also see if you have received a new exposure message.<br><br>If Smittestopp is inactive, you cannot receive messages. You can log in and share your status, but as there is no exchange of IDs when the app is inactive, there will be nobody to share your status with. <br><br><b>The technology</b> <br>The app uses Bluetooth and is based on built-in technology developed by Apple and Google. It comprises a system for notification of exposures. You can turn this on or off in your phone’s settings. In your phone’s settings, search for ‘covid’ or ‘smittestopp’ to turn the logging on or off. <br><br>Your phone generates a random Bluetooth ID every 15 minutes. When your phone is close to another app user’s phone, the two phones will exchange IDs.<br><br> Read more about the Smittestopp app at <a href=\\\\\"www.helsenorge.no/en/smittestopp\\\\\">www.helsenorge.no/en/smittestopp</a><br><br>\"",
```
Comment (nc-kano) [inline NDB.Covid19/NDB.Covid19/Locales/en.json:75]: still wrong url ;) backslashes and no https

File: NDB.Covid19/NDB.Covid19/Locales/nb.json [hunk 6]
```diff
@@ -73,17 +72,15 @@
...
... dele status i appen ved å gå til startsiden og trykke på «Har du testet positivt?». Andre brukere kan ikke se hvem du er. Du må logge deg inn via ID-porten.<br><br>Når du har delt, får andre brukere en melding om at de har vært i nærheten av en som har covid-19.<br><br>Du kan se meldinger i appen på startsiden. Her kan du også se om det er kommet en ny melding til deg.<br><br>Hvis Smittestopp ikke er aktiv, kan du ikke motta meldinger. Du må gjerne logge inn og dele status, men siden ID-er ikke blir utvekslet når appen ikke er aktiv, vil det ikke være noen å dele med. <br><br><b>Teknologien bak</b> <br>Appen bruker Bluetooth og er basert på en innebygd teknologi som er utviklet av Apple og Google. Teknologien er et system for varsling av smittekontakt. Du kan slå av og på denne varslingen under innstillingene på mobilen din. Under innstillingene kan du  søke på «covid» eller «smittestopp» for å slå av og på varslingen. <br><br>Mobilen din lager en tilfeldig Bluetooth-ID hvert 15. minutt. Når to mobiler er i nærheten av hverandre, utveksler de ID-er.<br><br> Les mer om Smittestopp-appen på <a href=\\\\\"www.helsenorge.no/smittestopp\\\\\">www.helsenorge.no/smittestopp</a><br><br>",
```
Comment (nc-kano) [inline NDB.Covid19/NDB.Covid19/Locales/nb.json:75]: url

File: NDB.Covid19/NDB.Covid19/Locales/nn.json [hunk 7]
```diff
@@ -73,17 +72,15 @@
...
...å til startsida og trykke på «Har du testa positivt?». Andre brukarar kan ikkje sjå kven du er. Du må logge deg inn via ID-porten.<br><br>Når du har delt statusen, vil andre brukarar få melding om at dei har vore i nærleiken av ein person som har covid-19.<br><br>Du kan sjå meldingar i appen på startsida. Her kan du òg sjå om du har fått nye meldingar.<br><br>Dersom Smittestopp ikkje er aktiv, kan du ikkje ta imot meldingar. Du kan gjerne logge deg inn og dele status, men sidan det ikkje blir utveksla ID-ar når appen ikkje er aktiv, vil du ikkje ha nokon å dele statusen med. <br><br><b>Teknologien bak</b> <br>Appen nyttar Bluetooth og er basert på ein innebygd teknologi som er utvikla av Apple og Google. Teknologien er eit system for varsling av smitte. Du kan slå av og på denne varslinga under innstillingane på mobilen din. Under innstillingane kan du søke på «covid» eller «smittestopp» for å slå av og på varslinga. <br><br>Mobilen din lagar ein tilfeldig Bluetooth-ID kvart 15. minutt. Når to mobilar er i nærleiken av kvarandre, utvekslar dei ID-ar.<br><br> Les meir om Smittestopp-appen på <a href=\\\\\"www.helsenorge.no/smittestopp\\\\\">www.helsenorge.no/smittestopp</a><br><br",
```
Comment (nc-kano) [inline NDB.Covid19/NDB.Covid19/Locales/nn.json:75]: url

**Review by nc-kano — APPROVED** [review]

**Pull request merged**